Assets/EngyneCreations/Trello/Scripts/TrelloBoardSettings.cs

```diff
@@ -14,53 +14,40 @@ public class TrelloBoardSettings : ScriptableObject
 		[Header("API Key and Token needed for Interacting with Trello")]
 		public string APIKey;
 		public string APIToken;
-
-		private static TrelloBoardSettings _instance;
-
-		public static TrelloBoardSettings Instance
-		{
-			get
-			{
-				LoadSettings();
-				return _instance;
-			}
-		}
-
-		public static void LoadSettings()
+		
+		public static TrelloBoardSettings LoadSettings()
 		{
-			_instance = FindOrCreateInstance();
+			return FindOrCreateInstance();
 		}
 
 		private static TrelloBoardSettings FindOrCreateInstance()
 		{
-			TrelloBoardSettings instance = null;
-			instance = Resources.Load<TrelloBoardSettings>("TrelloBoard");
-			instance = instance ? instance : Resources.LoadAll<TrelloBoardSettings>(string.Empty).FirstOrDefault();
-			instance = instance ? instance : CreateAndSave<TrelloBoardSettings>();
-
-			if (instance == null)
+			TrelloBoardSettings settings = null;
+			settings = Resources.Load<TrelloBoardSettings>("TrelloBoard") ?? CreateAndSave<TrelloBoardSettings>();
```

Comment:
`TrelloBoardSettings settings = Resources.Load<TrelloBoardSettings>("TrelloBoard") ?? CreateAndSave<TrelloBoardSettings>();`

And also, u don't need 2 methods that does the same thing. 
Just put this logic into `LoadSettings`